playbooks/generic/clean-obsolet-service.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Clean up obsolet service configuration
+  hosts: all
+  vars:
+    obsolet_service:
+    obsolet_service_os_name:
+    obsolet_containers_volumes:
+    obsolet_service_containers:
+    obsolet_service_dir:
+
+    gather_facts: false
+  tasks:
+    - name: "Stop/disable {{ obsolet_service }}-containers"
+      become: true
+      community.docker.docker_container:
+        name: "{{ item }}"
+        state: absent
+        stop_signal: true
+      with_items:
+        - "{{ obsolet_service_containers }}"
+
+    - name: "Stop/disable {{ obsolet_service }}-volumes"
+      become: true
+      community.docker.docker_volume:
+        name: "{{ item }}"
+        state: absent
+      with_items:
+        - "{{ obsolet_service }}"
+
+    - name: "Remove {{ obsolet_service }}-images"
+      become: true
+      community.docker.docker_image:
+         name: "{{ obsolet_service }}"
+         state: absent
```

Comment:
This will not work. Use a filter on the images. The image name != the service name in most cases.

Reply:
community.docker.docker_image not work switch to raw module to remove images